pyproject.toml

```diff
@@ -24,7 +24,7 @@ python = ">=3.6,<3.8"  # Compatible python versions must be declared here
 aws-requests-auth = "~0.4"
 boto3 = "~1.9"
 cryptography = "~2.5"
-django = "~2.2.7"
+django = "~2.2.9"
```

Comment:
In transmission this is set, with '==', to 2.2.12 as that is the latest 2.2 version without issues, and that is what is actually being installed. Does it still make sense to have it as "~2.2.9" instead of "==2.2.12" or even "~2.2"?

Reply:
In general, we want to have loose version pinning in python-common. We could consider pinning to ~2.2, but in this case I think we want the minimum version to be 2.2.9, in order to avoid these security issues.

Reply:
Whoops, this should be ~2.2.12, you're right.